tests/api/test_order.py

```diff
@@ -137,7 +139,27 @@ def test_order_query(
     assert expected_method.type.upper() == method['type']
 
 
-def test_order_events_query(
+def test_order_status_filter_param(user_api_client):
```

Comment:
Please add some asserts/checks to the test. I think we should have got fixtures for `ready_to_capture/fulfill` orders, if not, we can always create them.